src/locale/en_US.js

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+const en_US = {
+  datepicker: {
```

Comment:
could we co-locate the english labels with the component code? with the theme colors, i find it hard to jump between the different contexts. it's also tough to 'know' where these labels are located when coming to the project for the first time

Reply:
yeah, that makes sense